index.js

```diff
@@ -33,19 +34,24 @@ module.exports = function(userOpts) {
 		opts.dir = path.join(process.cwd(), opts.dir);
 	}
 
-	fs
-		.readdirSync(opts.dir)
+	fs.readdirSync(opts.dir)
 		.forEach(function(filename) {
+			gutil.log('[gulp-load-tasks]', filename.toString({ colors: true }));
+
 			var file = path.join(opts.dir, filename);
-			var stat = fs.statSync(file);
+			var fileExists = fs.existsSync(file);
```

Comment:
`fs.existsSync()` is deprecated (see https://nodejs.org/dist/latest/docs/api/fs.html#fs_fs_exists_path_callback)